### PR TITLE
New version: Waterfox.Waterfox version G6.0.11

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146305)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by aaronliu0130** [comment]
@CoolPlayLin @Exorcism0666 Change installer type to Nullsoft and add a displayversion per #146247

File: manifests/w/Waterfox/Waterfox/G6.0.11/Waterfox.Waterfox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Waterfox.Waterfox
+PackageVersion: G6.0.11
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 7.0.0.0
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/w/Waterfox/Waterfox/G6.0.11/Waterfox.Waterfox.installer.yaml:9]:
```suggestion
InstallerType: Nullsoft
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Change installer type to Nullsoft and add a displayversion per #146247

**Review by stephengillie — APPROVED** [review]

**Pull request merged**